File: logica-programacao/javascript/031.js
/*
Peça:
usuário
senha
Considere:
usuário correto = "admin"
senha correta = "1234"
Retorne True se usuário e senha forem corretos.
Caso contrário False. */ 

let usuario = prompt("Usuário: ").toLowerCase();
let senha = prompt("Senha: ");
let intSenha = parseInt(senha);

let verificao = usuario === "admin" && intSenha === 1234;
let fraseVerificacao = `Usuário e senha corretos?: ${verificao}`;

console.log(fraseVerificacao);

File: logica-programacao/python/031.py
# Peça:
# usuário
# senha
# Considere:
# usuário correto = "admin"
# senha correta = "1234"
# Retorne True se usuário e senha forem corretos.
# Caso contrário False.

usuario = input("Usuário: ").lower()
senha = input("Senha: ")
int_senha = int(senha)

verificacao = usuario == "admin" and int_senha == 1234
frase_verificao = f"Usuário e senha corretos?: {verificacao}"

print(frase_verificao)